developer-workflow/c-api.rst

```diff
@@ -4,18 +4,20 @@
 Changing Python's C API
 =======================
 
-The C API is divided into three sections:
+The C API is divided into these tiers:
 
 1. The internal, private API, available with ``Py_BUILD_CORE`` defined.
    Ideally declared in ``Include/internal/``. Any API named with a leading
    underscore is also considered private.
-2. The public C API, available when ``Python.h`` is included normally.
+2. The “general” public C API, available when ``Python.h`` is included normally.
```

Comment:
```suggestion
2. The "general" public C API, available when :cpy-file:`Include/Python.h` is included normally.
```
Should ASCII quotes be preferred? IIRC Sphinx already converts them to fancy quotes in the output.
Since this is the first time that `Python.h` is mentioned, it might be worth to include the full path and turn it into a link.

Reply:
> Should ASCII quotes be preferred? IIRC Sphinx already converts them to fancy quotes in the output.

Yeah I prefer them personally as they make find and replace easier, are slightly more efficient to store avoid accidentally copying the wrong quote and work equally well in code and prose text (with Sphinx formatting them appropriately), but as I've been trying to be less nitpicky about minor typographical details that don't make a different to the semantics or the output, I didn't mention it above.

>  Since this is the first time that Python.h is mentioned, it might be worth to include the full path and turn it into a link.

Yeah, good idea—I thought of that but didn't end up remembering to make the suggestion

Reply:
I strongly prefer using English quotes when writing English.

Reply:
As I see it, these arguments for `"` are exactly the same as arguments for one vs. two spaces after a period. For that there was a [recent decision](https://github.com/python/peps/pull/3027) to just use whatever the author prefers. So I'll do that. (I'm consciously trying to train my fingers to type the proper quotes for the language I'm writing: “English”, „Czech“, `'python'` or `"C"`.)
If you disagree, I don't think this PR is the right place for that discussion.

Reply:
> I strongly prefer using English quotes when writing English.

> (I'm consciously trying to train my fingers to type the proper quotes for the language I'm writing: “English”, „Czech“, `'python'` or `"C"`.)

Docutils/Sphinx will automatically convert standard ASCII straight quotes in the source files to the correct quote character for the configured document language (or leave them verbatim, for literal code blocks), so spending the extra effort of manually hardcoding the desired output typographical quote just bakes that complexity into the source and increases the probability of making a mistake (wrong quote char, wrong direction, etc) for no real practical gain.

That said, as mentioned above (and also in the discussions you linked), I don't see it as important enough to request changing in a review, much less argue about. :)

Reply:
```suggestion
2. The “general” public C API, available when :cpy-file:`Include/Python.h` is included normally.
```

The above substantive change without the quote change